CHANGELOG.md

```diff
@@ -28,7 +28,12 @@ Changelog
   
 - `QPU.get_version_info()` has been removed.
 
-- `QPU.get_calibration_program()` as been removed in favor of `QPU.calibration_program` (and optionally `QPU.refresh_calibration_program()`).
+- `get_qc()` now accepts an `execution_timeout` parameter (in addition to the existing `compiler_timeout`) to specify
+  a time limit on execution requests.
+
+- `QPUCompiler.get_calibration_program()` has been removed in favor of `QPUCompiler.calibration_program` (and optionally `QPUCompiler.refresh_calibration_program()`).
```

Comment:
Not technically in scope of this PR, but I made this correction while I was in here